spec/models/course_enrichment_spec.rb

```diff
@@ -0,0 +1,34 @@
+# == Schema Information
+#
+# Table name: course_enrichment
+#
+#  id                           :integer          not null, primary key
+#  created_by_user_id           :integer
+#  created_at                   :datetime         not null
+#  provider_code                :text             not null
+#  json_data                    :jsonb
+#  last_published_timestamp_utc :datetime
+#  status                       :integer          not null
+#  ucas_course_code             :text             not null
+#  updated_by_user_id           :integer
+#  updated_at                   :datetime         not null
+#
+
+require 'rails_helper'
+
+describe CourseEnrichment, type: :model do
```

Comment:
We've set up RSpec to infer types, not that I mind it being explicit.

Reply:
All the other specs specify type so I'd prefer to be consistent and remove type wholesale on a separate PR, if we want to do that.